src/main/java/com/spotlightspace/core/event/dto/AddEventRequestDto.java

```diff
@@ -0,0 +1,32 @@
+package com.spotlightspace.core.event.dto;
+
+import com.spotlightspace.core.event.domain.EventCategory;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AddEventRequestDto {
+    @NotBlank(message = "제목은 필수입니다.")
```

Comment:
```suggestion
@Getter
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@AllArgsConstructor
public class AddEventRequestDto {
    @NotBlank(message = "제목은 필수입니다.")
```
혹시나 만약 기본생성자를 사용하지 않으신다면 접근제어자를 private으로 바꾸는게 좋을 것 같아요!